llvm/test/Verifier/tbaa-struct.ll

```diff
@@ -0,0 +1,38 @@
+; RUN: llvm-as < %s 2>&1
+
+define void @test_overlapping_regions(ptr %a1) {
```

Comment:
might be good to add a FIXME that the verifier should reject the invalid `!tbaa.struct` nodes.

Reply:
Thanks, done!